library/data/tags/src/main/resources/fabric.mod.json

```diff
@@ -30,6 +34,14 @@
     "quilt_tags.mixins.json"
   ],
   "custom": {
+    "loom:injected_interfaces": {
+      "net/minecraft/class_3494": [
+        "org/quiltmc/qsl/tag/api/QuiltTag"
+      ],
+      "net/minecraft/class_3494\u0024class_3495": [
+        "org/quiltmc/qsl/tag/api/QuiltTagBuilder"
+      ]
+    },
```

Comment:
I'd appreciate it if we could somehow mark these interfaces as injected in the source code too. Maybe a marker annotation in `core`?

Reply:
The annotation `@InjectedInterface` has been added.